packages/server/modules/auth/tests/integration/registration.spec.ts

```diff
@@ -33,6 +37,22 @@ describe('Server registration', () => {
   let restApi: LocalAuthRestApiHelpers
   let apollo: TestApolloServer
 
+  const createInviteAsAdmin = async (
+    args: CreateProjectInviteMutationVariables | CreateServerInviteMutationVariables
+  ) => {
+    return await captureCreatedInvite(async () => {
```

Comment:
not a big deal, but as a general rule `return await` should be avoided

Reply:
Here's a reason for why I did it this way: https://jsfiddle.net/p2bz4j5x/

If you don't await the return, the parent function does not appear in the error stack trace

Reply:
![image](https://github.com/user-attachments/assets/828dd442-d1bc-4a0a-a071-28928b82a4fc)